src/app/subnets/views/SubnetDetails/StaticDHCPLease/ReserveDHCPLease/ReserveDHCPLease.test.tsx

```diff
@@ -76,6 +76,24 @@ it("displays an error if an out-of-range IP address is entered", async () => {
   ).toBeInTheDocument();
 });
 
+it("displays an error if the IP address or the MAC address are not entered", async () => {
+  state.subnet.items = [factory.subnet({ id: 1, cidr: "10.0.0.0/25" })];
+  renderWithBrowserRouter(
+    <ReserveDHCPLease
+      setSidePanelContent={vi.fn()}
+      subnetId={state.subnet.items[0].id}
+    />,
+    { state }
+  );
+
+  await userEvent.tab();
+  await userEvent.tab();
+  await userEvent.tab();
```

Comment:
This makes it a little more clear which fields are being acted on

```suggestion
  await userEvent.click(screen.getByRole("textbox", { name: "IP address" }));
  await userEvent.click(screen.getByRole("textbox", { name: "MAC address" }));
  await userEvent.tab();
```